src/components/context_menu/context_menu.test.tsx

```diff
@@ -155,6 +160,7 @@ describe('EuiContextMenu', () => {
         await tick(20);
 
         expect(takeMountedSnapshot(component)).toMatchSnapshot();
+        expect(onPanelChange).toHaveBeenCalledTimes(1);
```

Comment:
Oh actually, I totally forgot to mention this while we were pairing, but let's use `toHaveBeenCalledWith` to assert the contents.
```suggestion
        expect(onPanelChange).toHaveBeenCalledWith({
          panelId: 1,
          direction: 'previous',
        });
```